consensus/polybft/fsm.go

```diff
@@ -235,6 +250,32 @@ func (f *fsm) createBridgeCommitmentTx() (*types.Transaction, error) {
 	return createStateTransactionWithData(f.Height(), contracts.StateReceiverContract, inputData), nil
 }
 
+func (f *fsm) applySlashingTx() error {
+	if len(f.doubleSigners) == 0 {
+		return nil
+	}
+
+	slashingTx, err := f.createSlashingTx()
+	if err != nil {
+		return fmt.Errorf("creation of slashing transaction failed: %w", err)
+	}
+
+	if err = f.blockBuilder.WriteTx(slashingTx); err != nil {
+		return fmt.Errorf("failed to apply slashing state transaction. Error: %w", err)
```

Comment:
Nitpick (to be consistent with other error messages):
```suggestion
		return fmt.Errorf("failed to apply slashing state transaction: %w", err)
```

Reply:
Fixed [3a65efb](https://github.com/0xPolygon/polygon-edge/pull/1815/commits/3a65efb21cb39f9cc4f50c963d86c19959e114dd)